### PR TITLE
bump(*): source-to-image and k8s

### PR DESCRIPTION
**Pull request by adambkaplan**
Bumped dependencies:
github.com/openshift/source-to-image 27f0729
github.com/emicklei/go-restful-swagger12 5e28dc7
github.com/skynetservices/skydns 775ef406
k8s.io forks

* fixes [bug 1571349](https://bugzilla.redhat.com/show_bug.cgi?id=1571349)
* Support for ConfigMap Build Sources
    * [Trello Card](https://trello.com/c/RMKJxJUm/1020-5-allow-using-a-configmap-as-an-input-to-a-build-builds)
    * [RFE/bug 1540978](https://bugzilla.redhat.com/show_bug.cgi?id=1540978)
* Fix for [source-to-image#872](https://github.com/openshift/source-to-image/issues/872)

### CODE REVIEW

**Comment by bparees** [comment]
Commit text should be "bump(*)" since it's really bumping multiple deps.

Also I see the commits changed for some deps but no actual vendored files are changing for those packages, any idea what happened there?

File: glide.lock [hunk 1]
```diff
@@ -392,7 +392,7 @@ imports:
   subpackages:
   - log
 - name: github.com/emicklei/go-restful-swagger12
-  version: 76d80a40857969dddc1a4c21d1eb82f7ea20e9f5
+  version: 5e28dc7157ebd9247a268c26dd1bee7818aa33f7
```
Comment (liggitt) [inline glide.lock]: looks like this is undoing a bad update from https://github.com/openshift/origin/pull/19494/commits/1c853bdf4bf2e47ba6be53e18d38a95663fe9847 ... @smarterclayton any idea what happened on your PR? 76d80a40857969dddc1a4c21d1eb82f7ea20e9f5 doesn't exist, and 5e28dc7157ebd9247a268c26dd1bee7818aa33f7 is correct

File: vendor/github.com/google/cadvisor/manager/manager.go [hunk 2]
```diff
@@ -242,7 +242,7 @@ func retryDockerStatus() info.DockerStatus {
 	for {
 		ctx, _ := context.WithTimeout(context.Background(), startupTimeout)
 		dockerStatus, err := docker.StatusWithContext(ctx)
-		if err == nil {
+		if err != nil {
```
Comment (liggitt) [inline vendor/github.com/google/cadvisor/manager/manager.go]: this is reverting https://github.com/openshift/origin/pull/19601/files, drop this change

**Comment by liggitt** [comment]
> I see the commits changed for some deps but no actual vendored files are changing for those packages, any idea what happened there?

the kubernetes sha bumps mean it is picking up the latest shas the bot published out from openshift/origin -> openshift/kubernetes -> openshift/kubernetes-fork-repos. the fact that there are no diffs is a good sign... means the bot is keeping up

the cadvisor change with no sha bump means we're reverting an UPSTREAM that hasn't propagated out yet.

the emicklei/go-restful-swagger12 and skynetservices/skydns sha changes look right... @smarterclayton bumped deps a few days ago and it looks like bogus shas got pulled in with his PR... not sure how

**Comment by adambkaplan** [comment]
@liggitt ptal - reverted cadvisor change

**Comment by smarterclayton** [comment]
I might have a corrupted glide cache.

On May 4, 2018, at 2:42 PM, Jordan Liggitt <notifications@github.com> wrote:

I see the commits changed for some deps but no actual vendored files are
changing for those packages, any idea what happened there?

the kubernetes sha bumps mean it is picking up the latest shas the bot
published out from openshift/origin -> openshift/kubernetes ->
openshift/kubernetes-fork-repos. the fact that there are no diffs is a good
sign... means the bot is keeping up

the cadvisor change with no sha bump means we're reverting an UPSTREAM that
hasn't propagated out yet.

the emicklei/go-restful-swagger12 and skynetservices/skydns sha changes
look right... @smarterclayton <https://github.com/smarterclayton> bumped
deps a few days ago and it looks like bogus shas got pulled in with his
PR... not sure how

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub
<https://github.com/openshift/origin/pull/19627#issuecomment-386696942>,
or mute
the thread

**Comment by adambkaplan** [comment]
[(5) Allow using a ConfigMap as an input to a build [builds]](https://trello.com/c/RMKJxJUm/1020-5-allow-using-a-configmap-as-an-input-to-a-build-builds)

**Pull request merged**